frontend/public/components/overview/index.tsx

```diff
@@ -355,7 +355,7 @@ const OverviewHeading_: React.SFC<OverviewHeadingProps> = ({disabled, firstLabel
           Dashboard
         </button>
       </div>
-      <Toolbar className="overview-toolbar">
+      <Toolbar className="overview-toolbar" preventSubmit={true}>
```

Comment:
nit: this can be simply

```suggestion
      <Toolbar className="overview-toolbar" preventSubmit>
```

Reply:
fixed